.gitignore

```diff
@@ -0,0 +1,13 @@
+*-debug.log
+*-error.log
+.oclif.manifest.json
+/.nyc_output
+/lib
+/tmp
+/yarn.lock
+node_modules
+.DS_Store
+/test-results.xml
+coverage/
+junit.xml
+package-lock.json
```

Comment:
Please add `.env` as well, see my comment above.

Reply:
I removed the env file at all. This package should be used as a library(np, package), it shouldn't be configured by user in env way, as any npm package it should be installed and required parameters should be passed to lib constructor if required